shell/common/shell.h

```diff
@@ -271,6 +281,9 @@ class Shell final : public PlatformView::Delegate,
   uint64_t next_pointer_flow_id_ = 0;
 
   bool first_frame_rasterized_ = false;
+  std::atomic<bool> waiting_for_first_frame_ = true;
```

Comment:
Nit: `std::atomic_bool`

Reply:
std::atomic<bool> is used later in the file.